vyper/semantics/validation/utils.py

```diff
@@ -295,6 +293,29 @@ def types_from_UnaryOp(self, node):
         return _validate_op(node, types_list, "validate_numeric_op")
 
 
+def _is_empty_list(node):
+    """
+    Checks if a node is a `List` node with an empty list for `elements`,
+    including any nested `List` nodes.
+
+    Arguments
+    ---------
+    node: vy_ast.VyperNode
+        A Vyper node
+
+    Returns
+    -------
+    bool
+        Boolean value indicating if the node is an empty `List` node.
+    """
```

Comment:
since this is a private function we do not need to expose as a docstring
```suggestion
    # Checks if a node is a `List` node with an empty list for `elements`,
    # including any nested `List` nodes. ex. `[]` or `[[]]` will return True,
    # [1] will return False.
```